addons/60fps/addon.json

```diff
@@ -17,6 +17,14 @@
       "name": "TheColaber",
       "link": "https://scratch.mit.edu/users/TheColaber"
     }
+    {
+      "name": "GarboMuffin",
+      "link": "https://scratch.mit.edu/users/GarboMuffin"
+    }
+    {
+      "name": "TurboWarp by GarboMuffin",
+      "link": "https://turbowarp.org"
+    }
```

Comment:
Not needed. The credits only need people, not other things

Reply:
TurboWarp should be credited though